jenkins/internal-ci/generic/nightly/nightly-deploy.groovy

```diff
@@ -54,25 +55,41 @@ pipeline {
             }
         }
 
-        stage ("Deploy CORTX Cluster") {
+        stage ("Changesetlog generation") {
```

Comment:
`changelog generation` stage needs to be after `upload to github` stage. Changelog job fails to pull the latest job image as it's not pushed to GitHub yet. 
Refer - 
Nightly Job - https://eos-jenkins.colo.seagate.com/job/Cortx-Deployment/job/Nightly-K8s-Deploy/690/ 
Changelog Job - https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/Cortx-Automation/job/changelog-generation/366/